.github/workflows/sync-version.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  pull-request:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: pull-request
+      uses: repo-sync/pull-request@v2
+      with:
+        source_branch: ""                                 # If blank, default: triggered branch
+        destination_branch: "OpenFOAM7"                   
+        pr_title: "Synchronize ${{ github.ref }} with OpenFOAM7" # Title of pull request
+        pr_body: "*An automated PR*: additional branches to sync: OpenFOAM4, OpenFOAM6, OpenFAOMdev"              # Full markdown support, requires pr_title to be set
```

Comment:
This comment will quickly become outdated. Let's make it more generic or remove it once we implement the other jobs as well.

Actually, why not already implement everything else? It is a short copy-paste anyway. :D

Reply:
PRs are also about documentation and history. I am just a bit concerned that the history will be swamped by bot PRs.

Reply:
Then, we could convert this to a single issue or comment to an existing issue to update all branches. That would be a reminder and we would do a direct merge.

Reply:
Why not open up a PR as a working place? Then you have directly everything at hand..

Reply:
I am just proposing this as a fix to

> I am just a bit concerned that the history will be swamped by bot PRs.